js/script.js

```diff
@@ -0,0 +1,28 @@
+const taskInput = document.getElementById("task-input");
+const addButton = document.getElementById("btn-add");
+const unoderedList = document.getElementById("tasks");
+const notification = document.getElementById("message");
+
+let tasks = [];
+const addTaskHandler = ()=>{
+    const taskTitle = taskInput.value.trim();
+    if(taskTitle){
+        tasks.unshift(taskTitle);
+        const tasksList = document.createElement("li");
+        tasksList.textContent = taskTitle;
+        unoderedList.prepend(tasksList);
+        taskInput.value = "";
+    }
+    else
+        showMessage("Please provide valid Task.");
+};
+
+const showMessage = (message) => {
+    notification.textContent = message;
+    notification.hidden=false;
+    setTimeout(() => {
+        notification.hidden=true;
+    }, 1000);
+};
```

Comment:
It's better if you rename the function to showError or showErrorMessage for better readability. And inside the function you simply could have created a new HTML element to display the error message by setting a className and its textContent without using the setTimeout().

Reply:
Not only error messages but also success messages will also be shown via this element, that's why I think it's more convenient to keep the name as "showMessage". And used "setTimeout()" in order to remove the message after one second. I have also added the success message functionality.

https://github.com/sad-wtag/ToDoApp_VanillaJS/assets/164306406/8c0af656-ac54-4b8b-be43-fc2de21637d1